iac-extensions/docker/src/main/java/org/sonar/iac/docker/parser/grammar/DockerLexicalGrammar.java

```diff
@@ -168,7 +168,7 @@ private static void keywords(LexerlessGrammarBuilder b) {
       b.rule(tokenType).is(
         SPACING,
         b.regexp("(?i)" + tokenType.getValue()),
-        b.nextNot(b.regexp(LexicalConstant.IDENTIFIER))
+        b.nextNot(b.regexp("[^" + LexicalConstant.WHITESPACE + "\\\\]"))
```

Comment:
This can be simplified, there are two negations: `b.nextNot` and `[^` so it can be 
`b.regexp("[" + LexicalConstant.WHITESPACE + "\\\\]")`